File: FrontEnd/src/models/categoryVacanciesDTO.ts
export interface CategoryVacanciesDTO {
  id: string;
  status: number | null;
  createdDate: Date;
  positionName: string | null;
  numExp: number;
  degree: string | null;
  jobRequirements: string | null;
  jobDescription: string | null;
}

File: BackEnd/ExamManagement/DPD.HR.Business.Dto/Dto/CategoryVacanciesDto.cs
namespace DPD.HumanResources.Dtos.Dto;

public class CategoryVacanciesDto
{
    public Guid Id {get;set;}
    public int? Status {get;set;}
    public DateTime? CreatedDate {get;set;}
    public string? PositionName {get;set;}
    public float? NumExp {get;set;}
    public string? Degree {get;set;}
    public string? JobRequirements {get;set;}
    public string? JobDescription {get;set;}
}